docs/reference/remap/replace.cue

```diff
@@ -0,0 +1,73 @@
+package metadata
+
+remap: functions: replace: {
+	arguments: [
+		{
+			name:        "value"
+			description: "The original string."
+			required:    true
+			type: ["string"]
+		},
+		{
+			name:        "pattern"
+			description: "Replace all matches of this pattern."
+			required:    true
+			type: ["regex", "string"]
+		},
+		{
+			name:        "with"
+			description: "The string that the matches are replaced with."
+			required:    true
+			type: ["string"]
+		},
+	]
+	return: ["string"]
+	category: "text"
+	description: #"""
+			Replaces any matches of pattern with the provided string. Pattern can be either a fixed string or a regular expression.
+
+			Regular expressions take the form `/<regex>/<flags> where flags are one of the following:
+
+			- *i* perform a case insensitive match.
+			- *g* global. If specified all occurrences of the pattern are replaced. 
```

Comment:
`g` no longer exists (functions that can apply something 1 or more times have a `count` parameter added to them, wiht `-1` meaning "all")